src/code/templates/genome-challenge.js

```diff
@@ -30,10 +31,16 @@ export default class GenomeChallengeTemplate extends Component {
       onDrakeSubmission(targetDrake.phenotype.characteristics, userDrake.phenotype.characteristics, correct);
     };
 
-    const userDrakeStyle = userDrakeHidden ? "hiddenDrake" : "";
+    const instructionsBanner = instructions
+                                ? <div className="instructions-banner">
+                                    <div className="instructions-text">{instructions}</div>
+                                  </div>
+                                : null,
+          userDrakeStyle = !showUserDrake && userDrakeHidden ? "hiddenDrake" : "";
```

Comment:
`userDrakeHidden` was present before I became involved and appears to control the temporal visibility of a hidden drake after it has been submitted. I added 'showUserDrake` as a property that could be specified in authoring that describes a particular challenge.